src/skirmishbunker/bunkerBody.py

```diff
@@ -35,8 +36,8 @@ def make_interior_rectangle(self):
 
     self.interior_rectangle = (
         cq.Workplane("XY")
-        .box(self.int_length, self.int_width, self.height-self.wall_width)
-        .translate((0,0,self.wall_width/2))
+        .box(self.int_length, self.int_width, self.height)
+        .translate((0, 0, self.floor_thickness))
```

Comment:
this is setting the z-index offset in the positive direction.
because the box starts centered in the workplane, if you remove floor_thickness from the height of the box as I suggested you'll want to halve the distance traveled like I was doing with wall_width

Suggested when floor thickness is set:
.translate((0, 0, self.floor_thickness/2))